.github/workflows/ci.yml

```diff
@@ -183,6 +183,21 @@ jobs:
 
         sudo MKOSI_TEST_DEFAULT_VERB=boot python3 -m pytest -m integration -sv tests
 
+    # A single test for now should be sufficient
+    - name: Build/Boot ${{ matrix.distro }}/${{ matrix.format }} UsrOnly
+      if: ${{ matrix.distro == 'ubuntu' && matrix.format == 'gpt_ext4' }}
```

Comment:
Let's drop this condition and see how this fares on all distributions.

Reply:
Sure. Still keep it restricted to `gpt_ext4`?

Reply:
Let's try to run it for all formats, since the format actually makes for a noticeable difference here, e.g. as seen in the xfs PR

Reply:
OK, should I add another matrix option? I should've asked at the same time.

Reply:
No, that would take up too much CI capacity, and I'm suspecting booting usr only images is not going to work for most distros.

So I'd just keep it like you have now, but without the `if` and only try to boot in nspawn, not qemu. If booting fails for most distros, I'd only bother with trying to build the image and not booting it. We can add that later if we actually manage to fix booting distros with usronly